42mad

```diff
@@ -28,6 +28,15 @@ until /system/bin/curl -s -k -L -A "$useragent" -o "$2" "$1" ;do
 done
 }
 
+log_msg() {
+# $1 = severity
+# $2 = msg
+echo "$msg"
+if [ "$session_id" != "" ]; then
```

Comment:
im not using [ ] anywhere else in this script, use [[ ]] please.

Reply:
also, this should just be [[ "$session_id" ]]

Reply:
Updated in 0dc9c39